_posts/blog/2018-10-11-4rios-review.md

```diff
@@ -0,0 +1,30 @@
+tags: []
+author: "Kaitlin Burns"
+image:
+	feature:
+---
+# **Queering the Method: "4 Ríos" and Archival Narration**
+
+4 Ríos Digital Archive. http://4rios.co. Acccessed 28 Sept. 2018.
+
+The "4 Ríos" digital archive uses artful innovation to paint the indigenous and afro-descendant population of Colombia in the crossfire of armed conflict that has endured over more than forty years. This digital collection inspires the question, "What is an archive" as it reshapes notions on how information must be organized and presented. A truly multi-media experience, "4 Ríos" compiles video, testimony, art, and online sources in the larger framework of an interactive, graphic novel. Visitors to the archive participate in their own education regarding how the lives of Colombian people were spent like currency in the war between paramilitary and military forces, while witnessing information on the massacre at Alto de Naya in April 2001.
+
+Elder Manuel Tobar, an Audiovisual Producer and Social Activist by trade, created and directed the "4 Ríos" information collective in 2013. The description of his extensive team of specialists and their roles reads like movie credits, since the execution of this archive required unique talents such as actors, set designers, and story board creators. The project received the Premio Crea Digital in 2013 in recognition of its narrative value and commitment to social and political responsibility, and was funded through awards and grants from IDARTES (Instituto Distrital de las Artes) and Pontificia Universidad Javeriana in 2012, as well as through the Colombian Ministry of Culture in 2011.
+
+The stated goal of "4 Ríos" is to "propose other forms of approaching memory, to the construction of spaces of exchange and debate, to raise awareness about these facts, to not forgetting, to the rescue of that which, through the passing of time, is believed to be lost" ([Proyecto](http://4rios.co/blog/proyecto/), *my translation*). This is achieved by recounting experiences, facts, and the background and succession of evens in recent history of armed conflict in Colombia - specifically the 2001 massacre on the river Naya.
+
+On the opening webpage of "4 Ríos", the user's inital reaction is that of curious confusion, since the archive is not immediately recognizable as such. One might feel disillusioned by prior experiences with prototyical digital collections, library-like in their organization, as one attempts to master the interface and search for specific documentation. A visitor to "4 Ríos" transforms their experience when they agree to surrender instead, becoming a spectator as the narrative unfolds in graphic novel style, subject to the creators' conceived succession of information flow. If my description seems beyond grasp, it is because one must participate in this interactive historical retelling to conceive of its form.
```

Comment:
This is lovely. 

I'm not sure "disillusioned" is the right word. Maybe "far from prior experiences..."?

Reply:
do you think "estranged" works? i just don't want to get too wordy since it's already a bit of a long sentence...

Reply:
I like estranged!